cli/serve.go

```diff
@@ -23,6 +23,7 @@ func ServeCmd() *cobra.Command {
 		Use:           "serve",
 		Short:         "Start API server",
 		Long:          `Start API server`,
+		Args:          cobra.MaximumNArgs(1),
```

Comment:
no, `ExactArgs` or `MinimumNArgs` would make the argument mandatory. `MaximumNArgs` was chosen because it allows for `0` arguments.

The `serve` and `shell` subcommands currently accept no arguments so the existing behavior remains the untouched.

Reply:
see https://cobra.dev/#positional-and-custom-arguments for the docs

Reply:
NVM. I tested your PR and it works as intended.

Reply:
Yer, I misread `Max` as `Min` and jumped the gun :D